applications/osb-portal/src/pages/WorkspacePage.tsx

```diff
@@ -212,15 +216,25 @@ export const WorkspacePage = (props: any) => {
               <OSBSplitButton options={options} handleClick={openWithApp} />
             </Box>
           </Box>
+          {
+            /*
+              TODO: is this accordion being used? Looks like a duplicate of the left resource panel below
```

Comment:
@sanjayankur31 This accordion is used as a responsive alternative for small displays. You can see it by reducing the screen size